src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/RecalibrationEngine.java

```diff
@@ -33,11 +35,11 @@
      * @param covariates an array of the covariates we'll be using in this engine, order matters
      * @param numReadGroups the number of read groups we should use for the recalibration tables
      */
-    public RecalibrationEngine(final Covariate[] covariates, final int numReadGroups) {
+    public RecalibrationEngine(final StandardCovariateList covariates, final int numReadGroups) {
```

Comment:
All methods that take a `StandardCovariateList` as a parameter need to specify whether they expect the covariates to be initialized already, if covariate initialization remains optional. If covariate initialization is made mandatory in `StandardCovariateList`, then you can ignore this comment.